src/main/java/seedu/address/model/policy/Policy.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.policy;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+/**
+ * Represents a Person's policy in the client list.
+ * Guarantees: policyName and description are present and not null,
+ * field values are validated, immutable.
+ */
+public class Policy {
+
+    private final PolicyName policyName;
+    private final PolicyDescription description;
+
+    /**
+     *
+     * @param policyName
+     * @param description
```

Comment:
Perhaps we should add in javadoc? Same for the rest of this class.

Reply:
Changed this. thanks for pointing it out